boskos/cmd/boskos/boskos.go

```diff
@@ -68,6 +69,8 @@ func init() {
 func main() {
 	logrusutil.ComponentInit("boskos")
 	kubeClientOptions.AddFlags(flag.CommandLine)
+	var namespace string
+	flag.StringVar(&namespace, "namespace", corev1.NamespaceDefault, "namespace to install on")
```

Comment:
is there a reason you put this in main() instead of with the rest of the flags above?

Reply:
Oversight, fixed